AppCenter/AppCenter/MSConstants.h

```diff
@@ -44,6 +44,11 @@ typedef NS_ENUM(NSUInteger, MSLogLevel) {
   MSLogLevelNone = 99
 };
 
+/**
+ * Default flush interval for channel.
+ */
+static NSUInteger const MSFlushIntervalDefault = 3.0;
```

Comment:
You also need a `k` prefix by convention on the constant name.
```suggestion
static NSUInteger const kMSFlushIntervalDefault = 3.0;
```